.classpath

```diff
@@ -1,17 +1,18 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="lib" path="distro/lib/jcurses-x.jar"/>
-	<classpathentry kind="lib" path="distro/lib/libjcsi.jar"/>
-	<classpathentry combineaccessrules="false" kind="src" path="/Serf"/>
-	<classpathentry kind="lib" path="distro/lib/basicPlayer/jl1.0.jar"/>
-	<classpathentry kind="lib" path="distro/lib/basicPlayer/jorbis-0.0.15.jar"/>
-	<classpathentry kind="lib" path="distro/lib/commons-httpclient-3.0.1.jar"/>
-	<classpathentry kind="lib" path="distro/lib/basicPlayer/commons-logging-api.jar"/>
-	<classpathentry kind="lib" path="distro/lib/basicPlayer/tritonus_share.jar"/>
-	<classpathentry kind="lib" path="distro/lib/basicPlayer/javazoom3.0-openJDK-modfix-src.zip"/>
-	<classpathentry kind="lib" path="distro/lib/basicPlayer/vorbisspi1.0.2.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/3"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="distro/lib/jcurses-x.jar"/>
+	<classpathentry kind="lib" path="distro/lib/libjcsi.jar"/>
+	<classpathentry kind="lib" path="distro/lib/basicPlayer/jl1.0.jar"/>
+	<classpathentry kind="lib" path="distro/lib/basicPlayer/jorbis-0.0.15.jar"/>
+	<classpathentry kind="lib" path="distro/lib/commons-httpclient-3.0.1.jar"/>
+	<classpathentry kind="lib" path="distro/lib/basicPlayer/commons-logging-api.jar"/>
+	<classpathentry kind="lib" path="distro/lib/basicPlayer/tritonus_share.jar"/>
+	<classpathentry kind="lib" path="distro/lib/basicPlayer/javazoom3.0-openJDK-modfix-src.zip"/>
+	<classpathentry kind="lib" path="distro/lib/basicPlayer/vorbisspi1.0.2.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>

```

Comment:
As far as I can see, is this the only change on the .classpath ? (changing to v 4), as well as adding log4j ?